File: 김강민/10월/Week_02/1016/12930.py
def solution(s):
    answer = ''
    s_spl = s.split(' ')

    for i in s_spl:
        for j in range(len(i)):
            if j % 2 == 0:
                answer += i[j].upper()
            else:
                answer += i[j].lower()
        answer += ' '
    return answer[0:-1]

File: 구은성/11월/1105/12930.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] letters = s.split("");
        int idx = 0;
        
        for(String letter : letters) {
            if(letter.equals(" ")) {
                answer += " ";
                idx = -1;
            }
            else if(idx % 2 == 0) {
                answer += letter.toUpperCase();
            }
            else {
                answer += letter.toLowerCase();
            }
            
            idx++;
        }
        
        return answer;
    }
}